apiml/src/test/java/org/zowe/apiml/ZaasSchemeTransformApiTest.java

```diff
@@ -46,294 +41,321 @@ class ZaasSchemeTransformApiTest {
     private AuthSourceService authSourceService;
     private PassTicketService passTicketService;
     private ZaasSchemeTransformApi transformApi;
+    TokenCreationService tokenCreationService;
+    ZosmfService zosmfService;
+    RequestCredentials credentials;
+    AuthSource authSource;
     private final MessageService messageService = new YamlMessageService("/apiml-log-messages.yml");
 
+    private final String INVALID_AUTH_MSG = "ZWEAO402E The request has not been applied because it lacks valid authentication credentials.";
+    private final String MISSING_AUTH_MSG = "ZWEAG160E No authentication provided in the request";
+
     @BeforeEach
     void setUp() {
         authSourceService = mock(AuthSourceService.class);
+        when(authSourceService.isValid(any())).thenReturn(true);
+        tokenCreationService = mock(TokenCreationService.class);
+        zosmfService = mock(ZosmfService.class);
         passTicketService = mock(PassTicketService.class);
-        ZosmfService zosmfService = mock(ZosmfService.class);
-        TokenCreationService tokenCreationService = mock(TokenCreationService.class);
-
+        credentials = mockCredentials();
+        authSource = mock(AuthSource.class);
         transformApi = new ZaasSchemeTransformApi(
             authSourceService,
             passTicketService,
-            zosmfService,
+            mock(ZosmfService.class),
```

Comment:
```suggestion
            zosmfService,
```
Or is it intentional?

Reply:
probably mistake